src/main/java/com/datadoghq/sketch/ddsketch/encoding/VarEncodingHelper.java

```diff
@@ -12,6 +12,37 @@
   private static final int MAX_VAR_LEN_64 = 9;
   private static final int VAR_DOUBLE_ROTATE_DISTANCE = 6;
 
+  private static final byte[] UNSIGNED_VAR_LONG_LENGTHS = new byte[65];
+  private static final byte[] VAR_DOUBLE_LENGTHS = new byte[65];
+
+  static {
```

Comment:
can we get a comment for this block?

Reply:
Added in f0956fe8a9711b09a76e770e6af351c7f75b9268.